openverse_catalog/dags/providers/provider_api_scripts/freesound.py

```diff
@@ -6,8 +6,10 @@
 Output:                 TSV file containing the image, the respective
                         meta-data.
 
-Notes:                  https://freesound.org/apiv2/search/text'
-                        No rate limit specified.
+Notes:                  https://freesound.org/apiv2/search/text.
+                        Rate limit is specified here (2k requests/day):
+                        https://freesound.org/docs/api/overview.html#throttling.
+                        Maximum page size is 150.
```

Comment:
Can we undo this change to avoid conflict with #928?

Reply:
Oh shoot, yes! Good callout!